### ISSUE TITLE
Bulk codefixes are broken

### ISSUE THREAD

**Issue by psfinaki** [dotnet/fsharp#14967]
C#: 

https://user-images.githubusercontent.com/5451366/227215548-b842d0f6-7b88-448f-836a-78b5b5ca1f7b.mp4

F#:

https://user-images.githubusercontent.com/5451366/227216995-6a3db9e4-d2b2-467d-b65a-75f8e19c7138.mp4

Nothing happens in F#. This is sad.

**Known workarounds**

Apply codefixes one by one? Regex replace? 

**Additional context**

Discovered during some F# session with @T-Gro

**Comment by T-Gro** [comment]
Found something in the docs and we do not seem to use it centrally.

```csharp
namespace Microsoft.CodeAnalysis.CodeFixes

open Microsoft.CodeAnalysis.CodeFixes


///<summary>
///            Implement this type to provide fixes for source code problems.
///            Remember to use <see cref="T:Microsoft.CodeAnalysis.CodeFixes.ExportCodeFixProviderAttribute" /> so the host environment can offer your fixes in a UI.
///            </summary>
type CodeFixProvider =
  
  new: unit -> unit
  
  member ComputeRequestPriority: unit -> Microsoft.CodeAnalysis.CodeActions.CodeActionRequestPriority
  
  ///<summary>
  ///            Gets an optional <see cref="T:Microsoft.CodeAnalysis.CodeFixes.FixAllProvider" /> that can fix all/multiple occurrences of diagnostics fixed by this code fix provider.
  ///            Return null if the provider doesn't support fix all/multiple occurrences.
  ///            Otherwise, you can return any of the well known fix all providers from <see cref="T:Microsoft.CodeAnalysis.CodeFixes.WellKnownFixAllProviders" /> or implement your own fix all provider.
  ///            </summary>
  member GetFixAllProvider: unit -> FixAllProvider

```


The unused opens uses a wellknown batch provider, which has limitation in it's remarks which causes the unused opens to not work.
I have a feeling that the limitation might have been added later after F# implemented this codefix.. ?

```csharp
///<summary>
///            Contains well known implementations of <see cref="T:Microsoft.CodeAnalysis.CodeFixes.FixAllProvider" />.
///            </summary>
type WellKnownFixAllProviders =
  
  ///<summary>
  ///            Default batch fix all provider.
  ///            This provider batches all the individual diagnostic fixes across the scope of fix all action,
  ///            computes fixes in parallel and then merges all the non-conflicting fixes into a single fix all code action.
  ///            This fixer supports fixes for the following fix all scopes:
  ///            <see cref="F:Microsoft.CodeAnalysis.CodeFixes.FixAllScope.Document" />, <see cref="F:Microsoft.CodeAnalysis.CodeFixes.FixAllScope.Project" />, <see cref="F:Microsoft.CodeAnalysis.CodeFixes.FixAllScope.Solution" /><see cref="F:Microsoft.CodeAnalysis.CodeFixes.FixAllScope.ContainingMember" /> and <see cref="F:Microsoft.CodeAnalysis.CodeFixes.FixAllScope.ContainingType" />.
  ///            </summary>
  ///<remarks>
  ///            The batch fix all provider only batches operations (i.e. <see cref="T:Microsoft.CodeAnalysis.CodeActions.CodeActionOperation" />) of type
  ///            <see cref="T:Microsoft.CodeAnalysis.CodeActions.ApplyChangesOperation" /> present within the individual diagnostic fixes. Other types of
  ///            operations present within these fixes are ignored.
  ///            </remarks>
  static member BatchFixer: FixAllProvider

```

**Comment by 0101** [comment]
Duplicate of https://github.com/dotnet/fsharp/issues/14864
